spec.html

```diff
@@ -7598,7 +7598,7 @@
         <emu-alg>
           1. Let _args_ be the arguments object.
           1. If _P_ is `"caller"`, then
-            1. Let _desc_ be ! OrdinaryGetOwnProperty(_args_, _P_).
+            1. Let _desc_ be OrdinaryGetOwnProperty(_args_, _P_).
             1. If IsDataDescriptor(_desc_) is *true*, return *true*.
```

Comment:
Per 5.2 all abstract operations return completion records. Either ! or ? must be used to unwrap a value (although we retain for now the semantics where we can implicitly treat a completion record as its unwrapped value in certain circumstances).